File: 2221.数组的三角和.go
/*
 * @lc app=leetcode.cn id=2221 lang=golang
 *
 * [2221] 数组的三角和
 */

// @lc code=start
func triangularSum(nums []int) int {
	for n := len(nums) - 1; n > 0; n-- {
		for i := 0; i < n; i++ {
			a[i] = (a[i] + a[i+1]) % 10
		}
	}
	return a[0]
}

// @lc code=end

File: 2221.数组的三角和.py
#
# @lc app=leetcode.cn id=2221 lang=python3
#
# [2221] 数组的三角和
#

# @lc code=start
class Solution:
    def triangularSum(self, nums: List[int]) -> int:
        for j in range(len(nums) - 1, 0, -1):
            for i in range(j):
                nums[i] = (nums[i] + nums[i+1]) % 10
        return nums[0]
# @lc code=end

